src/models/importer.ts

```diff
@@ -0,0 +1,42 @@
+import { EventEmitter } from "events";
+import fs from "fs";
+import Papa from "papaparse";
+import { promisify } from "util";
+
+export class Importer {
+  eventEmitter: EventEmitter;
+  async: boolean;
+
+  constructor (eventEmitter: EventEmitter, async: boolean = true) {
```

Comment:
In typescript there's no need for assignment to class fields
If you need to assign smth in constructor, you can just do that

```
constructor (private eventEmitter: EventEmitter, private async: boolean = true) { // empty constructor }
```

And also don't forget about the descriptors, if you're not sure which one to use, read the docs or contact me and I'llk explain